text/2018-08-29-destroy-range.md

```diff
@@ -0,0 +1,50 @@
+# Summary
+
+Support RPC `DestroyRange`. This call is on the whole TiKV rather than a certain region. When it is invoked, TiKV will use `delete_files_in_range` to quickly free a large amount of space and delete all keys in the range. Raft layer will be bypassed, and the range can cover across multiple regions. The invoker should promise that after invoking `DestroyRange`, the range will never be accessed again. That is to say, the range is permanently scrapped.
+
+This interface is only designed for TiDB.
```

Comment:
This interface currently is only designed for TiDB,  it is used when drop/truncate a huge table.